src/commands/admin/warn.js

```diff
@@ -0,0 +1,37 @@
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { MessageEmbed, Permissions } from 'discord.js';
+
+export const data = new SlashCommandBuilder()
+	.setName('warn')
+	.setDescription('Warns a user.')
+	.addUserOption((option) =>
+		option.setName('user').setDescription('The user to warn.').setRequired(true)
+	);
+
+export async function execute(interaction, client) {
+	const userService = client.database.UserService;
+
+	if (!interaction.member.permissions.has(Permissions.FLAGS.ADMINISTRATOR)) {
+		await interaction.reply({
+			content: "You don't have permission to do that!",
+			ephemeral: true,
+		});
+		return;
+	}
+
+	let data = await userService.fetchUser(interaction.user.id);
```

Comment:
data is referring to `src/database/user.service.js` and the idea in Klara was to have a reputation system that would follow all servers with the bot, getting negative reputation on warns and positive on reactions on messages. A positive warn value should be added to `src/database/guild.service.js` and a reason

Reply:
A guild id should be added indeed, Nick should not transfer reputation over to another server.